src/app/shared/wysiwyg/prosemirror.d.ts

```diff
@@ -1,149 +1,3 @@
-  export { chainCommands }
-  const baseKeymap: any;
-  export { baseKeymap }
-}
-
-declare module 'prosemirror-commands/platform' {
-  const mac: boolean;
-  export { mac }
-  const ios: boolean;
-  export { ios }
-}
-
-declare module 'prosemirror-keymap' {
-  export { keymap } from 'prosemirror-keymap/keymap';
-}
-
-declare module 'prosemirror-keymap/keymap' {
-  const keymap: any;
-  export { keymap }
-}
-
-// incomplete
-declare module 'prosemirror-history' {
-  export { HistoryState, history } from 'prosemirror-history/history';
-}
-
-declare module 'prosemirror-history/history' {
-  class HistoryState {
-    constructor(done: any, undone: any, prevMap: any, prevTime: any);
-  }
-  export { HistoryState }
-  const history: any;
-  export { history }
-}
-
-declare module 'prosemirror-inputrules' {
-  export { InputRule, inputRules } from 'prosemirror-inputrules/inputrules';
-  export { emDash, ellipsis, openDoubleQuote, closeDoubleQuote, openSingleQuote, closeSingleQuote, smartQuotes, allInputRules } from 'prosemirror-inputrules/rules';
-  export { wrappingInputRule, textblockTypeInputRule, blockQuoteRule, orderedListRule, bulletListRule, codeBlockRule, headingRule } from 'prosemirror-inputrules/util';
-}
-
-declare module 'prosemirror-inputrules/inputrules' {
-  class InputRule {
-    constructor(match: any, handler: any);
-  }
-  export { InputRule };
-  function inputRules(ref: any): any;
-  export { inputRules };
-}
-
-declare module 'prosemirror-inputrules/rules' {
-  import { InputRule } from 'prosemirror-inputrules';
-  const emDash: InputRule;
-  export { emDash };
-  const ellipsis: InputRule;
-  export { ellipsis };
-  const openDoubleQuote: InputRule;
-  export { openDoubleQuote };
-  const closeDoubleQuote: InputRule;
-  export { closeDoubleQuote };
-  const openSingleQuote: InputRule;
-  export { openSingleQuote };
-  const closeSingleQuote: InputRule;
-  export { closeSingleQuote };
-  const smartQuotes: InputRule[];
-  export { smartQuotes };
-  const allInputRules: InputRule[];
-  export { allInputRules };
-}
-
-declare module 'prosemirror-inputrules/util' {
-  import { InputRule } from 'prosemirror-inputrules';
-  function wrappingInputRule(regexp: any, nodeType: any, getAttrs?: any, joinPredicate?: any): InputRule;
-  export { wrappingInputRule };
-  function textblockTypeInputRule(regexp: any, nodeType: any, getAttrs?: any): InputRule;
-  export { textblockTypeInputRule };
-  function blockQuoteRule(nodeType: any): InputRule;
-  export { blockQuoteRule };
-  function orderedListRule(nodeType: any): InputRule;
-  export { orderedListRule };
-  function bulletListRule(nodeType: any): InputRule;
-  export { bulletListRule };
-  function codeBlockRule(nodeType: any): InputRule;
-  export { codeBlockRule };
-  function headingRule(nodeType: any, maxLevel: any): InputRule;
-  export { headingRule };
-}
-
 declare module 'prosemirror-markdown' {
```

Comment:
for some reason I don't understand, it doesn't see the exported schema from @types/prosemirror-markdown, so I had to keep just our type defs for prosemirror-markdown